apps/bare-expo/ios/Podfile.lock

```diff
@@ -3633,7 +3633,7 @@ SPEC CHECKSUMS:
   fast_float: 06eeec4fe712a76acc9376682e4808b05ce978b6
   FBLazyVector: 73f927157a20d4c20c419d3ea42197c28e895fc3
   fmt: a40bb5bd0294ea969aaaba240a927bd33d878cdd
-  glog: eb93e2f488219332457c3c4eafd2738ddc7e80b8
+  glog: 5683914934d5b6e4240e497e0f4a3b42d1854183
```

Comment:
Whats this?

Reply:
After an upgrade sometimes the lock file needs to updated. Removing node_modules and pods and reinstalling everything verifies if the change is necessary